packages/desktop-client/src/locales/en-GB.json

```diff
@@ -11,5 +11,23 @@
     "title": "Bootstrap this Actual instance",
     "setPassword": "Set a password for this server instance",
     "tryDemo": "Try Demo"
+  },
+  "schedules": {
+    "account": "Account",
```

Comment:
Some of these translations are quite general and so we'll likely use them across the app, e.g. `Account`, `Amount`, `Payee`.

We probably want to pull them up out of the `schedules` scope to sit at the top level.

Reply:
@TomAFrench I agree. What key do you suggest? Something like "general."?

Reply:
Ideally we'd just have them without a prefix but that'll cause a collision on `account` as it is. `general.` is fine for now as we can easily restructure this file in future.

Reply:
On my way 🏃‍♂️💨

Reply:
@TomAFrench Take a look to the keys. Is very difficult to draw a line.